detekt-gradle-plugin/src/main/kotlin/io/gitlab/arturbosch/detekt/invoke/CliArgument.kt

```diff
@@ -80,6 +83,14 @@ internal data class BasePathArgument(val basePath: String?) : CliArgument() {
     override fun toArgument() = basePath?.let { listOf(BASE_PATH_PARAMETER, it) }.orEmpty()
 }
 
+internal data class FailOnSeverityArgument(val ignoreFailures: Boolean, val minSeverity: FailOnSeverity) :
+    CliArgument() {
+    override fun toArgument(): List<String> {
+        val effectiveSeverity = if (ignoreFailures) FailOnSeverity.NEVER else minSeverity
+        return listOf(FAIL_ON_SEVERITY_PARAMETER, effectiveSeverity.name.toLowerCase(Locale.ROOT))
```

Comment:
```suggestion
        return listOf(FAIL_ON_SEVERITY_PARAMETER, effectiveSeverity.name.lowercase())
```
🤔 Isn't this the not deprecated one?

Reply:
Yes it is. Maybe you can help me on this. For some reason the `lowercase` extension function is not available in the detekt-cli main source set. It is available in the test source set though.

Reply:
I assume that the gradle plugin code uses an older version of kotlin. Nothing to do here then.